product_analytic/README.rst

```diff
@@ -0,0 +1,52 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+================
+Product Analytic
+================
+
+This module allows you to configure an **income analytic account** and an **expense analytic account** on products and on product categories. When you select the product in an invoice line, it will check if this product has an income analytic account (for customer invoice/refunds) or an expense analytic account (for supplier invoice/refunds) ; if it doesn't find any, it checks if the category of the product has an income or expense analytic account ; if an analytic account is found, it will be set by default on the invoice line.
```

Comment:
You can put this in the Usage section, and put here
```
This module allows to define an analytic account at product or category level for using it when creating invoices.
```
And please respect 80 cols limit also in this file.